CONTRIBUTING.md

```diff
@@ -1,91 +1,214 @@
 # Contributing to Connectors
 
-When you want to contribute to Connectors, we highly suggest you include tests.
+## Prerequisites
 
-To test connectors, you'll need your own account and instance with the provider you're testing (like Salesforce or Hubspot). You'll also need to set up credentials (using oAuth2.0) to link the connectors to the providers.
+1. To test connectors, you'll need your own developer account and instance with the provider you're testing (like Salesforce or Hubspot).  Follow the provider guide on how to do this [here.](https://docs.withampersand.com/provider-guides/overview)
 
-Since each provider has different credential parameters and everyone has different environments, we've made a `CredentialRegistry` and `Reader` interface to make credential loading consistent. Please use these. We currently support `JSONReader`, `EnvVarReader`, and `ValueReader`.
+2. You'll need to obtain the credentials (using oAuth2.0 or apiKey) to link the connectors to the providers. 
 
-For examples on how to do this, check out the tests in `test/salesforce` and `test/hubspot`.
+3. Add a `creds.json` at the root of the connectors repo. Since each provider has different credential parameters and everyone has different environments, we've made a `CredentialRegistry` and `Reader` interface to make credential loading consistent. Please use these. We currently support `JSONReader`, `EnvVarReader`, and `ValueReader`.
```

Comment:
```suggestion
3. Add a `creds.json` file at the root of the `connectors` repo. Since each provider has different credential parameters and different environments, we've made a `CredentialRegistry` and `Reader` interface to make credential loading consistent which we recommend using for an easier development experience. We currently support `JSONReader`, `EnvVarReader`, and `ValueReader`.
```